entry_tasks/89_sequence_type_check/main.py

```diff
@@ -0,0 +1,49 @@
+def parse_value(string_value):
+    if string_value.isdigit():
+        return int(string_value)
+    elif string_value.strip().replace('.', '').isdigit():
+        return float(string_value)
+    elif string_value.strip().lower() == "false":
+        return False
+    elif string_value.strip().upper() == "False":
+        return False
+    elif string_value.strip().lower() == "true":
+        return True
+    elif string_value.strip().upper() == "True":
+        return True
+    elif string_value.strip().lower() == "none":
+        return None
+    elif string_value.strip().upper() == "None":
+        return None
+    elif string_value.__str__():
+        if "(" in string_value and ")" in string_value:
+            array_tuple = string_value.replace("(", "").replace(")", "").replace(" ", "").split(",")
+            for idx, item in enumerate(array_tuple):
+                array_tuple[idx] = parse_value(item)
+            return tuple(array_tuple)
+        else:
+            return str(string_value.strip())
+    elif string_value.isspace():
+        return string_value.replace(" ", "")
+
+
+if __name__ == "__main__":
+    lst = []
+    a = input("Input some sequence values, separated by comma: ")
+
+    start_index = 0
+    while start_index < len(a):
+        comma_index = a.find(',', start_index)
+        if comma_index == -1:
+            value = a[start_index:]
+        else:
+            value = a[start_index:comma_index]
+            start_tuple_index = value.find('(')
+            if start_tuple_index != -1:
+                end_tuple_index = a.find(')', start_index)
+                value = a[start_index:end_tuple_index + 1]
+        start_index = start_index + len(value) + 1
+        value = value.strip()
+        parsed_value = parse_value(value)
+        lst.append((parsed_value, type(parsed_value)))
+    print(lst)
```

Comment:
try to input just a blank space

` `
or 
`  ,  `

I think blank space is still a string, but not None technically speaking

Reply:
Sorry, didn't get you here:( What do you exactly mean?

Reply:
> @mykytapavlov Sorry, didn't get you here:( What do you exactly mean?

Already got it)
Code has been modified.

Reply:
Better, but now try to input:
`1, ` with a blank space after comma

Reply:
@mykytapavlov Done.

Reply:
`1,  , 2` ? :)

But we can close it really, overall idea is covered :)
Overall, task was to ignore blank spaces so I agree that this case can be invalid.